src/staging/src/pages/404.js

```diff
@@ -1,46 +1,42 @@
 import * as React from "react"
-import { Link } from "gatsby"
 
-import BetaBanner from "../components/betaBanner"
+import Header from "../components/header"
 import Footer from "../components/footer"
 import Seo from "../components/seo"
 import ThumbLink from "../components/thumbLink"
 
-import BCParksLogo from "../images/bcparks-h-rgb-rev.png"
-
 import "../styles/404.scss"
 
 const NotFoundPage = () => (
   <>
-    <BetaBanner></BetaBanner>
     <Seo title="404: Not found" />
+    <Header />
     <div className="not-found">
-      <nav className="navbar navbar-dark" id="desktopNav">
-        <div className="container">
-          <div className="p-md-3 p-sm-0 mx-2">
-            <Link to="/">
-              <img className="bc-parks-logo" height="50" src={BCParksLogo} alt="BC Parks logo" />
-            </Link>
-          </div>
-        </div>
-      </nav>
       <div>
         <div className="not-found-container d-flex align-items-center justify-content-center container">
           <div className="p-3">
             <div className="mx-2">
-              <h1 className="text-white">This page is still being designed.<br /><br />
-                Please visit our featured pages below.</h1>
+              <p>
+                Good catch! You seem to have found a link that’s not working or a page that hasn’t been added to our new
+                website yet. Help us fix things as soon as possible by letting us know at{" "}
+                <a href="mailto:parkinfo@gov.bc.ca" aria-label="Send a mail to BC Parks">
+                  parkinfo@gov.bc.ca
+                </a>
+              </p>
+              <p>
+                You may find what you were looking for at the links below.
+              </p>
             </div>
             <div className="mt-2 d-flex justify-content-center">
               <div className="row">
                 <div className="col-lg-4 col-md-12 mt-2">
                   <ThumbLink imageLink="404/home-thumb.jpg" title="Homepage" navLink="/" />
                 </div>
                 <div className="col-lg-4 col-md-12 mt-2">
-                  <ThumbLink imageLink="404/park-thumb.jpg" title="Park Search" navLink="/find-a-park" />
+                  <ThumbLink imageLink="404/park-thumb.jpg" title="Find a park" navLink="/find-a-park" />
```

Comment:
The Thumblink component is unnecessarily fussy. If you're in the mood, I'd rather you just take it out completely since this is the only place we use it, and it's more complex than we need.

If that looks like it might cause issues, it's ok to leave it.

Reply:
It might be useful to keep the `Thumblink` component since we can pass three different props. We could map the image element but assigning each link and title to each image element requires us to make some functions. So I'd prefer to keep it for the code readability. However, it's true that the `Thumblink` component is kinda fussy so I deleted unnecessary code.

Reply:
Makes sense, thanks!